test/src/middlewares/childLogger.test.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+/* eslint no-unused-expressions:0 */
+
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const childLogger = require('../../../src/middlewares/childLogger.js');
+
+describe('child logger middleware - childLogger.js', function root() {
```

Comment:
You can use an arrow function at describe level if you don't use mocha's feature for name and timeout manipulations.